torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -899,6 +900,29 @@ RegisterOperators reg(
              return 0;
            };
          }),
+     Operator(
+         prim::sort,
```

Comment:
Why not just register this with schema, and then list aliasing behavior in the schema to be conservative.